source/common/config/filter_json.cc

```diff
@@ -236,10 +236,10 @@ void FilterJson::translateRedisProxy(
   proto_config.set_cluster(json_config.getString("cluster_name"));
 
   const auto json_conn_pool = json_config.getObject("conn_pool");
-  json_conn_pool->validateSchema(Json::Schema::REDIS_CONN_POOL_SCHEMA);
```

Comment:
v1 config is being removed soon, so please revert the changes to this file and config_schemas.cc